docs/client-side-scripting.md

```diff
@@ -5,7 +5,7 @@ found and run in the app.
 
 Why do this?
 
-- GitHub Pages does not support JS bundles natively, and will block any
+- GitHub Pages do not support JS bundles natively, and will block any
```

Comment:
I believe this is correct as it refers to GitHub Pages (the product) rather than a plural number of pages

Reply:
That makes sense. My apologies I read it in the context of actual number of pages!